### ISSUE TITLE
Rails 3.1 compatibility

### ISSUE THREAD

**Issue by contentfree** [sporkd/compass-html5-boilerplate#19]
I'm sure we're all interested in hearing how Compass & the HTML5 Boilerplate work with the new asset pipeline in Rails 3.1. Could we has info?

**Comment by sporkd** [comment]
Yea, I have not started tackling this yet because it will really depend on how compass handles things going forward.  I know that the compass team is hard at work with the Rails core team to make compass work seamlessly with sprockets and tilt.  And I have been tracking this issue from the start https://github.com/chriseppstein/compass/issues/337

**Comment by andresgutgon** [comment]
Hello. Now / Rails 3.1.rc4 and compass are compatible no? this compass-html5-boilerplate is compatible or is very soon? :)
The README is not compatible with rails 3.1

**Comment by sporkd** [comment]
Yes, you are right.  Due to the large differences with how compass now integrates with rails and the asset pipeline, I  have decided to start a new project.  This also allows me make some better design decisions such as separating the rails gem from the compass stuff.  Also, the new version is now an engine which will soon have generators to copy views that you want to override.

I have not updated the readme here yet because it's a little to soon to be useable for just anyone, but you're welcome to give it a test spin...  https://github.com/sporkd/html5-rails

Just keep in mind that you will manually need to copy any view from the gem's app/views directory you want to override.  And, there may be non-backwards compatible changes coming soon.

**Comment by andresgutgon** [comment]
Many, many thanks for your response. I´ll give a try

**Comment by andresgutgon** [comment]
how can i know when html5-rails is ready? In your blog feeds o where?

**Comment by sporkd** [comment]
I'll post to the readme here on the is repo.  To get it working now, you may also need to remove your application.html.erb and your application.js from your rails 31 project. This will let the engine take over.  Then you can start copying those files back over from the gem to customize.

**Comment by andresgutgon** [comment]
Thanks! 
I´ll follow the project. Congrats!

**Comment by digitalplaywright** [comment]
I'am interested in transitioning to rails 3.1 and would like to continue using compass-html5-boilerplate. Any updates to the status of the rails 3.1 support is greatly appreciated.

**Comment by kucaahbe** [comment]
maybe [this](http://kucaahbe.github.com/2011/08/30/html5-boilerplate-and-rails3.1.html) will help someone

**Comment by coffeesam** [comment]
Hi, is there an update to the Rails 3.1 compatibility? Is it working now?

**Comment by sporkd** [comment]
Hey coffeesam.  For Rails 3.1, the html5-rails gem is still the way to go.  I just added a skimpy readme that will get you going for the time being.

https://github.com/sporkd/html5-rails

**Comment by coffeesam** [comment]
Hi Peter,

Ok cool, thanks alot for info.

Cheers,
Sam

On Thu, Dec 8, 2011 at 4:04 PM, Peter Gumeson <
reply@reply.github.com
